mm2src/coins/eth.rs

```diff
@@ -5807,15 +5877,26 @@ impl MmCoin for EthCoin {
                 }
                 gas
             },
-            EthCoinType::Nft { .. } => return MmError::err(TradePreimageError::NftProtocolNotSupported),
+            // TODO update err type
```

Comment:
i think this todo is resolved ha?

p.s. otherwise, needs more explanation.

Reply:
its a leftover, thanks, removed it https://github.com/KomodoPlatform/komodo-defi-framework/pull/2467/commits/1f69c9d0e62b5e8f996f29f04c50a51a2e6baec8